### PR TITLE
🐛 EXA: update circulating supply calculation

### PR DESCRIPTION
**Pull request by franm91**

### CODE REVIEW

File: pages/api/circulating-exa.ts [hunk 1]
```diff
@@ -21,11 +26,73 @@ const EXCLUDED_ADDRESSES = [
...
+    )) as { streams: Stream[] };
+
+    streams.push(...data.streams);
+    last = data.streams.length ? data.streams[data.streams.length - 1].id : undefined;
+  } while (last);
+  return streams;
+}
+
+const withdrawable = async () => {
+  const allStrams = await allAccounts();
```
Comment (sebipap) [inline pages/api/circulating-exa.ts:64]: typo

File: pages/api/circulating-exa.ts [hunk 2]
```diff
@@ -21,11 +26,73 @@ const EXCLUDED_ADDRESSES = [
   airdrop,
 ];
 
-const exa = {
```
Comment (sebipap) [inline pages/api/circulating-exa.ts:24]: nit: why case change? it's not constant

File: pages/api/circulating-exa.ts [hunk 3]
```diff
@@ -39,13 +106,14 @@ export default async function (_: NextApiRequest, res: NextApiResponse) {
...
-        { ...exa, functionName: 'totalSupply' },
-        ...EXCLUDED_ADDRESSES.map((address) => ({ ...exa, functionName: 'balanceOf', args: [address] })),
+        { ...EXA, functionName: 'decimals' },
+        { ...EXA, functionName: 'totalSupply' },
+        ...EXCLUDED_ADDRESSES.map((address) => ({ ...EXA, functionName: 'balanceOf', args: [address] })),
       ],
     });
     const nonCirculatingSupply = balancesResult.reduce((total, { result }) => total + (result as bigint), 0n);
-    const circulatingSupply = (totalSupply as bigint) - nonCirculatingSupply;
+    const totalWhitdrawable = await withdrawable();
```
Comment (sebipap) [inline pages/api/circulating-exa.ts:115]: typo

File: pages/api/circulating-exa.ts [hunk 4]
```diff
@@ -39,13 +106,14 @@ export default async function (_: NextApiRequest, res: NextApiResponse) {
...
+    const circulatingSupply = (totalSupply as bigint) - nonCirculatingSupply + totalWhitdrawable;
```
Comment (sebipap) [inline pages/api/circulating-exa.ts:116]:
```suggestion
    const circulatingSupply = BigInt(totalSupply) - nonCirculatingSupply + totalWhitdrawable;
```

**Review by itofarina — APPROVED** [review]

**Review by sebipap — APPROVED** [review]

**Pull request merged**